### PR TITLE
fix(Resource-status): Various fixes

### PR DESCRIPTION
**Pull request by bdauria**
## Description

This fixes:
- The columns configuration not being saved when leaving the page
- The possibility to unselect all columns (at least one must be selected)
- The Resources with only passive checks not being displayed in the "checks" column
- The visibility of Resources with enabled notification in the "netofiication" column
- Tre information column collapsing (minimum width set)

**Fixes** # (issue)

## Type of change

- [x] Patch fixing an issue (non-breaking change)

## Target serie

- [x] 21.04.x (master)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "@babel/preset-env": "^7.13.12",
     "@babel/preset-react": "^7.12.13",
     "@babel/preset-typescript": "^7.13.0",
-    "@centreon/frontend-core": "github:centreon/frontend-core",
+    "@centreon/frontend-core": "github:centreon/frontend-core#6079299b5e4b7608813bfa40df90f724ace6bbe9",
```
Comment (Thebarda) [inline package.json:27]: Will you take off this commit hash in another PR ?

**Review by Thebarda — APPROVED** [review]

File: www/front_src/src/Resources/Listing/columns/Checks.tsx [hunk 2]
```diff
@@ -1,22 +1,45 @@
...
+  if (row.active_checks === false) {
     return (
-      <IconColumn>
-        <Tooltip title={labelChecksDisabled}>
-          <SyncDisabledIcon color="primary" fontSize="small" />
-        </Tooltip>
-      </IconColumn>
+      <Column Icon={SyncProblemIcon} title={t(labelOnlyPassiveChecksEnabled)} />
     );
   }
```
Comment (kduret) [inline www/front_src/src/Resources/Listing/columns/Checks.tsx:45]: If passive checks and active checks are both enabled, no icon will be displayed ?
Is it what we expert ?

Comment (bdauria) [reply www/front_src/src/Resources/Listing/columns/Checks.tsx:45]: Yes

**Review by kduret — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (bdauria) [reply package.json:27]: Yes, I’ll wait that all the current PRs are merged so that we don’t deal with gigantic conflicts

**Pull request merged**